### DISCUSSION TITLE
How do I get a log of all requests made to OpenAI?

### DISCUSSION THREAD

**Discussion by 7ammer** [FlowiseAI/Flowise#966] [Q&A]
I'd like to see exactly what is being sent to OpenAI. I'm using the sql database chain and I'd like to see what data is being sent. DEBUG does not show me this.

**Comment by HenryHengZJ** [comment]
Click analyse chatflow,  we support multiple obeservability platforms - 
<img width="479" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/26460777/8b801be4-73dd-4b41-8e8c-6eba54919642">
